test/polytopes.jl

```diff
@@ -934,5 +934,47 @@
       ├─ Point(0.0, 1.0, 0.0)
       └─ Point(0.0, 0.0, 1.0)"""
     end
+
+    # measure
+    # Points forming a square with the first point in the center.
+    pts = [P3(0, 0, 0), P3(-1, -1, 0), P3(-1, 1, 0), P3(1, 1, 0), P3(1, -1, 0)]
+    simplicies = connect.([(1, 2, 3), (1, 3, 4), (1, 4, 5), (1, 5, 2)], Simplex)
```

Comment:
Please avoid using the `connect` here as it is unrelated to the construction of `Simplex` or the `measure` test. The constructor can be used directly.

Reply:
Done.